### ISSUE TITLE
Option to disable processing

### ISSUE THREAD

**Issue by ProLoser** [bdkjones/CodeKit-1-Old#386]
I want to process my SASS, but not my JS. Our build server is capable of doing certain processing but not others.

Instead of having to manually disable processing for each and every file individually, it would be nice if I could disable it for the project as a whole. It's annoying that my only options are 'Concat' and 'Concat + minify'. Can you simply add a 'Disabled' option to these lists?

**Comment by iangilman** [comment]
I thought this too, but I think Concat doesn't do anything if there are no files to import, so that's your Disabled option there. Still, I agree it's confusing.

**Comment by ProLoser** [comment]
What do you mean no files to import? As in code kit import tags?

**Comment by iangilman** [comment]
Exactly. So as long as they're just normal JavaScript files with no import tags, nothing should happen (beyond linting).

**Comment by Azuka** [comment]
I'm having the same issue. I still get errors every time I edit a js file. Options are **Check Syntax with Nothing** and **Concatenate imported files** but for some reason I still get JSHint errors even when I set the file not to compile directly or hide it.

I think we need an option that explicitly marks a js or all js files as not monitored by CodeKit.

**Comment by bmatzner** [comment]
+1
